lib/matplotlib/stackplot.py

```diff
@@ -23,32 +23,35 @@ def stackplot(axes, x, *args, **kwargs):
 
     *x* : 1d array of dimension N
 
-    *y* : 2d array of dimension MxN, OR any number 1d arrays each of dimension
-          1xN. The data is assumed to be unstacked. Each of the following
-          calls is legal::
+    *y* :
+        2d array of dimension MxN, OR any number 1d arrays each of dimension 1xN.
+        The data is assumed to be unstacked.
+        Each of the following calls is legal::
 
             stackplot(x, y)               # where y is MxN
             stackplot(x, y1, y2, y3, y4)  # where y1, y2, y3, y4, are all 1xNm
 
     Keyword arguments:
 
-    *baseline* : ['zero', 'sym', 'wiggle', 'weighted_wiggle']
-                Method used to calculate the baseline. 'zero' is just a
-                simple stacked plot. 'sym' is symmetric around zero and
-                is sometimes called `ThemeRiver`.  'wiggle' minimizes the
-                sum of the squared slopes. 'weighted_wiggle' does the
-                same but weights to account for size of each layer.
-                It is also called `Streamgraph`-layout. More details
-                can be found at http://leebyron.com/streamgraph/.
+    *baseline* :
```

Comment:
Yep… We have this everywhere in our code, but this is a mistake that we keep propagating :(
There should be no markup here :)